cmds/cpud/serve_test.go

```diff
@@ -34,14 +37,20 @@ func TestListen(t *testing.T) {
 		{"tcp", "17010"},
 		{"tcp4", "17010"},
 		{"tcp6", "17010"},
-		// Damn. Can't easily test this b/c environments vary so much.
-		//{"vsock", "17010"},
+		{"vsock", "17010"},
 		{"unix", "@foobar"},
 	}
 
 	for _, tt := range oktests {
 		ln, err := listen(tt.network, tt.port)
 		if err != nil {
+			var sys_err *os.SyscallError
```

Comment:
nit: use `sysErr` instead of `sys_err` to follow the convention in Go

Reply:
done